packages/renderer/src/lib/preferences/PreferencesConnectionCreationOrEditRendering.spec.ts

```
@@ -392,21 +382,21 @@ describe.each([
     const createButton = screen.getByRole('button', { name: `${label}` });
     expect(createButton).toBeInTheDocument();
     await vi.waitFor(() => expect(createButton).toBeDisabled());
 
     await fireEvent.input(inputElement as Element, { target: { value: '2' } });
-    await vi.waitFor(() => expect(vi.mocked(window as any).auditConnectionParameters).toBeCalledTimes(2));
+    await vi.waitFor(() => expect(vi.mocked(window.auditConnectionParameters)).toBeCalledTimes(2));
     await vi.waitFor(() => expect(createButton).toBeEnabled());
   });
 });
 
 test(`Check itemsAudit receive updated values`, async () => {
```

Biome (2.5.6)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)